packages/boundless-checkbox-group/style.styl

```diff
@@ -1,5 +1,8 @@
 @require "variables"
-@require "node_modules/boundless-checkbox/style";
+
+unless flat-build {
```

Comment:
so what this is doing is ignoring the `@require` statement when we're building the master skin because the glob import will grab it already